### PR TITLE
Added react app

### PR DESCRIPTION
**Pull request by willrodriguez**
This is a react based coding interview.

### CODE REVIEW

File: enterprise/client-list/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* `icon-rl-search` search icon for search box.
+
+![](mock.png)
+
+#### Bonus points
+
+* Make the alphapicker filter functional.
+* Make the search box filter functional.
+* Make the product & status filters collapsible.
+* Make the product & status filters functional.
```
Comment (Tuizi) [inline enterprise/client-list/README.md:36]: + ADA Compliant (ScreenReader, Keyboard navigation etc...)

Comment (willrodriguez) [reply enterprise/client-list/README.md:36]: Updated

File: enterprise/client-list/src/index.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Unify Sans', Arial, Helvetica, sans-serif
+}
+
+code {
```
Comment (Tuizi) [inline enterprise/client-list/src/index.css]: Remove this block?

Comment (willrodriguez) [reply enterprise/client-list/src/index.css]: block has been removed.

**Pull request merged**